### PR TITLE
Minor style changes to interfaces docs

### PR DESCRIPTION
**Pull request by almarklein**
issues: none

@bryevdv asked via flowdock to check on http://bokeh.pydata.org/en/latest/docs/user_guide/plotting.html a short while ago. I opened the page to look at it soon, only to find a week later, with the PR already merged.

Anyway, it's a very nice overview IMO. Here are some small style issues.

### CODE REVIEW

File: sphinx/source/docs/user_guide/interfaces.rst [hunk 1]
```diff
@@ -78,8 +78,8 @@ Bokeh provides a "mid-level" general purpose |bokeh.plotting| interface, which
 is similar in specificity to Matplotlib_ or Matlab_ style plotting interfaces.
 It is centered around having users relate the visual glyphs they would like
 to have displayed to their data, and otherwise taking care of putting together
-plots with sensible default axes, grids, and tools. All of the hard work to
-assemble the appropriate Bokeh Models in the correct way to form a scenegraph
```
Comment (bryevdv) [inline sphinx/source/docs/user_guide/interfaces.rst]: Not opposed to some change here, but this was intentional -- I am trying to highlight that the act of assembling using the low level glyphs is not trivial.

File: sphinx/source/docs/user_guide/interfaces.rst [hunk 2]
```diff
@@ -19,22 +19,22 @@ browsers for presentation providing elegant, concise construction of novel
...
-Offering both powerful and flexible features to enable very advanced
+To offer both powerful and flexible features, enabling advanced
 customizations on one hand and simplicity on the other, Bokeh exposes
 different interface levels to the users:
 
 *bokeh.models*
 --------------
 
-Bokeh is actually composed of two library components.
+Bokeh is composed of two library components.
```
Comment (bryevdv) [inline sphinx/source/docs/user_guide/interfaces.rst]: Also open to revision, but this was also intentional, to highlight the dichotomy itself, not just state it.

**Comment by bryevdv** [comment]
@almarklein when making in "issue PR" please make sure it it tagged, etc., according to https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management

**Comment by bryevdv** [comment]
These look good, further feedback appreciated on two points.

File: sphinx/source/docs/user_guide/interfaces.rst (hunk 1 above)
Comment (almarklein) [inline sphinx/source/docs/user_guide/interfaces.rst]: I'll put "hard" back then.

**Comment by almarklein** [comment]
Ah I seem to have forgotten the tags, sorry about that. I reverted the change w.r.t. the two components.

**Pull request merged**